web_widget_image_download/__openerp__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Flavio Corpa <flavio.corpa@tecnativa.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+{
+    "name": "Web Widget - Image Download",
+    "summary": "Allows to download the avatar for the contact",
```

Comment:
Actually the "this module was written to ..." part is more for the README, and lately just removed (all are written modules after all). However something like "Allows to download any image from its widget" seems better.